### PR TITLE
Fixes #369 - incorrect spelling

### LINKED ISSUE #369: Incorrect spelling of "regularly"
The following files have the misspelling:

- [markdown.test.js.snap](https://github.com/rahuldkjain/github-profile-readme-generator/tree/master/src/components/__tests__/__snapshots__/markdown.test.js.snap)
- [markdownPreview.test.js.snap](https://github.com/rahuldkjain/github-profile-readme-generator/tree/master/src/components/__tests__/__snapshots__/markdownPreview.test.js.snap)
- [markdown.test.js](https://github.com/rahuldkjain/github-profile-readme-generator/tree/master/src/components/__tests__/markdown.test.js)
- [markdownPreview.test.js](https://github.com/rahuldkjain/github-profile-readme-generator/tree/master/src/components/__tests__/markdownPreview.test.js)

There was a commit previously to fix [the typo](https://github.com/rahuldkjain/github-profile-readme-generator/commit/ec78c6ba0534beec8ff4915f94d0e9adbd199b5c#diff-26ca35c3be6d07c84bb77f1e63a1b19487279d5b9c9a9392179b3a35647e1254L45) in a commit [here](https://github.com/rahuldkjain/github-profile-readme-generator/commit/ec78c6ba0534beec8ff4915f94d0e9adbd199b5c#diff-2715e47e84094ab6a942260599168f35875404bea9e9c38760595e94528ab2aaR12). I assume the test cases got missed out.

### PR DESCRIPTION
**Pull request by aravindvnair99**
Fixes #369

Changed `regulary` -> `regularly` as it's a typo.

### CODE REVIEW

File: src/components/__tests__/__snapshots__/markdown.test.js.snap [hunk 1]
```diff
@@ -54,7 +54,7 @@ exports[`Markdown renders devDynamicBlogs is true 1`] = `
   />
   <DisplayWork
     link="blog"
-    prefix="📝 I regulary write articles on"
+    prefix="📝 I regularly  write articles on"
```
Comment (harshcut) [inline src/components/__tests__/__snapshots__/markdown.test.js.snap:57]: @aravindvnair99 It looks like you have added a double space after `regularly` in every correction.

Comment (aravindvnair99) [reply src/components/__tests__/__snapshots__/markdown.test.js.snap:57]: @harshcut Ah yes, thanks for noticing. I hadn't realised that. I've fixed it now.

**Review by harshcut — APPROVED** [review]
LGTM 👍

**Comment by aravindvnair99** [comment]
@rahuldkjain I believe this PR can be merged.

**Comment by g-savitha** [comment]
please close this pr @aravindvnair99

**Comment by aravindvnair99** [comment]
> please close this pr @aravindvnair99

@g-savitha It gets closed automatically when you merge.

**Pull request merged**